upload/admin/controller/startup/language.php

```diff
@@ -3,7 +3,7 @@
 class Language extends \Opencart\System\Engine\Controller {
 	public function index() {
 		// Language
-		$query = $this->db->query("SELECT * FROM `" . DB_PREFIX . "language` WHERE code = '" . $this->db->escape($this->config->get('config_admin_language')) . "'");
+		$query = $this->db->query("SELECT * FROM `" . DB_PREFIX . "language` WHERE `code` = '" . $this->db->escape($this->config->get('config_admin_language')) . "'");
```

Comment:
catalog/controller/startup/startup.php file uses models to pull languages. The admin-end side should do the same.